clients/include/testing_scal_batched_ex.hpp

```diff
@@ -109,20 +112,53 @@ hipblasStatus_t testing_scal_batched_ex_template(Arguments argus)
 
         // enable unit check, notice unit check is not invasive, but norm check is,
         // unit check and norm check can not be interchanged their order
-        if(argus.unit_check)
+        if(unit_check)
         {
             unit_check_general<Tx>(1, N, batch_count, incx, hz, hx);
         }
 
     } // end of if unit check
 
-    //  BLAS_1_RESULT_PRINT
+    if(timing)
+    {
+        hipStream_t stream;
+        status = hipblasGetStream(handle, &stream);
+        if(status != HIPBLAS_STATUS_SUCCESS)
+        {
+            hipblasDestroy(handle);
+            return status;
+        }
+
+        int runs = argus.cold_iters + argus.iters;
+        for(int iter = 0; iter < runs; iter++)
+        {
+            if(iter == argus.cold_iters)
+                gpu_time_used = get_time_us_sync(stream);
 
+            status = hipblasScalBatchedExFn(
+                handle, N, &alpha, alphaType, dx, xType, incx, batch_count, executionType);
+
+            if(status != HIPBLAS_STATUS_SUCCESS)
+            {
+                hipblasDestroy(handle);
+                return status;
+            }
+        }
+        gpu_time_used = get_time_us_sync(stream) - gpu_time_used;
+
+        ArgumentModel<e_N, e_alpha, e_incx, e_batch_count>{}.log_args<Tx>(
+            std::cout,
+            argus,
+            gpu_time_used,
+            scal_gflop_count<Tx, Ta>(N),
+            scal_gbyte_count<Tx>(N),
+            hipblas_error);
```

Comment:
I noticed that we don't calculate `hipblas_error` anywhere. This should be calculated using `norm_check_general` in the `if(unit_check)` section. I notice it's missing in `testing_scal.hpp` as well, might be missing in other places as well but that can be a different PR.